src/Microsoft.ML.Data/Transforms/ValueToKeyMappingEstimator.cs

```diff
@@ -46,13 +46,13 @@ public abstract class ColumnOptionsBase
 
             [BestFriend]
             private protected ColumnOptionsBase(string outputColumnName, string inputColumnName,
-                int maxNumKeys, SortOrder sort, string[] term, bool textKeyValues)
+                int maxNumberOfKeys, SortOrder sort, string[] term, bool textKeyValues)
```

Comment:
>sort [](start = 47, length = 4)

suggestion: sortOrder
sort sounds like a boolean on whether to sort or not.

Reply:
I went for MappingOrder


---
In reply to: [262809389](https://github.com/dotnet/machinelearning/pull/2862#discussion_r262809389) [](ancestors = 262809389)